캘린더 수정 샘플2

stomimh opened the pull request.

stomimh committed:
캘린더 수정 샘플2

kongyi123 commented on temp3/app/src/main/java/com/example/temp/ui/theme/CalendarAdpt.kt:
```diff
@@ -0,0 +1,79 @@
+
+        public  fun setCalendar(position:Int)
+        {
+            var Year=cal.get(Calendar.YEAR)+((position+cal.get(Calendar.MONTH))/12-100)
```
var Year=cal.get(Calendar.YEAR)+((position-1200 +cal.get(Calendar.MONTH))/12)
// 의미 단위로 따로 구분이 가능하게하여 readibility를 높이는 것이 좋습니다.

Young-in-Kong committed:
fixed the issue showing the wrong date as today.

Young-in-Kong committed:
fix to get permission for SMS and get automatically get the phone number of the device.
- to get the phone number of the device, SMS and phone read permission was needed. (it was changed)
fix the history list to just show recent 100 items.

Young-in-Kong committed:
2024 -> 2025

Young-in-Kong committed:
2025 -> 2026

Young-in-Kong committed:
fix to work alarm function
